common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -421,6 +422,7 @@ class @Problem
 
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
+        @enableSubmitButton false, false
```

Comment:
Why is the submit button being disabled here? Reloading the page re-enables the submit button, as does changing the selected answer. I don't see the point in disabling it on "Show Answer".